gpustat/core.py

```diff
@@ -719,3 +719,26 @@ def new_query():
     to get the list of GPUs and running process information.
     '''
     return GPUStatCollection.new_query()
+
+
+def gpu_count() -> int:
+    '''
+    Return the number of GPUs in the system.
+    '''
+    try:
+        N.nvmlInit()
+        return N.nvmlDeviceGetCount()
+    except N.NVMLError:
+        return 0  # fallback
```

Comment:
Or maybe let `gpu_count()` fail here and catch the exception in function `is_available()`.